faasmcli/faasmcli/tasks/flame.py

```diff
@@ -12,12 +12,12 @@
 
 
 @task(default=True)
-def general(ctx, user, func, reps=10, cmd=None, data=None, reverse=False):
+def general(ctx, user, func, cmd=None, data=None, reverse=False):
```

Comment:
Including multiple repetitions in the runner scripts was only really useful for profiling, and added needless complexity otherwise.